### PR TITLE
Add version constant

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

**Comment by saicheems** [comment]
@jba 

For some background:
For each language we send an `x-goog-api-agent` header along with requests. It's purpose is to track the codegen version and gax version used with each gapic API call. Since other languages have package management systems which require version labels, they can retrieve this at runtime.

We don't... so this is my proposal. It may not be ideal, but please let me know if there is a better way to do this.

**Comment by saicheems** [comment]
@pongad 
@vchudnov-g 
@jmuk

File: gax.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package gax
+
+const (
+	Version = "0.10.0"
```
Comment (jmuk) [inline gax.go]: out of curiosity: why 10?

**Comment by jmuk** [comment]
LGTM, I was just curious when I wrote the comment above.

File: gax.go (hunk 1 above)
Comment (saicheems) [inline gax.go]: Err yeah mean't 0.1.0
Thanks, fixed

File: gax.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package gax
+
+const (
```
Comment (jba) [inline gax.go]: remove parens

**Comment by jba** [comment]
I think this is fine.

There is another question, which is what happens if there is an incompatible change to the gax package. Do the clients vendor gax? I don't think they should, but then changes to gax could break all the clients. Anyway, this isn't a worry for this PR.

**Comment by saicheems** [comment]
This is a good point I haven't considered...
I'd prefer the clients not have to vendor, but if there are major changes, there's not an easy way around that

File: gax.go (hunk 2 above)
Comment (saicheems) [inline gax.go]: Done

**Pull request merged**